tests/unit/Util/Xml/ValidatorTest.php

```diff
@@ -52,4 +52,23 @@ public function testDoesNotValidateInvalidXmlFile(): void
         $this->assertTrue($result->hasValidationErrors());
         $this->assertSame(PHP_EOL . '  Line 17:' . PHP_EOL . '  - Element \'filter\': This element is not expected.' . PHP_EOL, $result->asString());
     }
+
+    public function testDoesNotThrowExceptionWithDisabledEntityLoader(): void
+    {
+        libxml_disable_entity_loader(true);
+
+        $result = (new Validator())->validate(
+            (new Loader)->loadFile(
+                __DIR__ . '/../../../end-to-end/migration/possibility-to-migrate-from-92-is-detected/phpunit.xml',
+                false,
+                true,
+                true
+            ),
+            (new SchemaFinder)->find(Version::series())
+        );
+
+        $this->assertTrue($result->hasValidationErrors());
+        $this->assertSame(PHP_EOL . '  Line 17:' . PHP_EOL . '  - Element \'filter\': This element is not expected.' . PHP_EOL, $result->asString());
+        $this->assertTrue(libxml_disable_entity_loader(true), 'libxml_disable_entity_loader must be set back to it\'s original value');
```

Comment:
To just "read" the state, we do not need to set "true".

Reply:
I'm not aware of a read-only function to get the current libxml_disable_entity_loader-state. Die functions default value is `true`.

Reply:
At least according to the manual, the parameter is optional. So I at least assume that would be for "reading" the current state. I of course didn't try that ;)

Reply:
Unfortunately not, see: https://3v4l.org/NJprJ
 
```php
var_dump(libxml_disable_entity_loader(false)); // init with false
var_dump(libxml_disable_entity_loader()); // "read" value
var_dump(libxml_disable_entity_loader()); // value was modified
```

Reply:
\o/ Okay, never mind. *sigh*